cmd/jujud/agent/machine/manifolds.go

```diff
@@ -816,25 +768,6 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 			NewFacade:     credentialvalidator.NewFacade,
 			NewWorker:     credentialvalidator.NewWorker,
 		}),
-
-		legacyLeasesFlagName: ifController(featureflag.Manifold(featureflag.ManifoldConfig{
-			StateName: stateName,
-			FlagName:  feature.LegacyLeases,
-			Logger:    loggo.GetLogger("juju.worker.legacyleasesenabled"),
-			NewWorker: featureflag.NewWorker,
-		})),
-
-		certificateUpdaterName: ifFullyUpgraded(certupdater.Manifold(certupdater.ManifoldConfig{
```

Comment:
do we need this for `CaaS` controller?

Reply:
It won't work because it watches machine addresses.
It's only relevant for HA (not implemented yet on k8s). And for k8s we use a service with a stable IP address anyway